File: 2405.1/comchk/c/FieldComment.cpp
#include <iostream>

using namespace std;

class People {
private:
	// int 타입 변수: 나이
	int age;
	/* string 타입 변수: 이름 */
	string name;
	/* string 타입 변수: 
	 * 주소 
	 */
	string address;

public:
	string getName() {
        return this->name;
    }
    void setName(string _name) {
        this->name = _name;
    }
};

File: 2405.1/comchk/java/FieldComment.java
package com.sparrow;

import java.lang.String;

public class People {
    // int 타입 변수: 나이
    private int age;
    /* String 타입 변수: 이름 */
    private String name;
    /* String 타입 변수: 
	 * 주소 
	 */
    private String address;

    public String getName() {
        return this.name;
    }
    public void setName(String name) {
        this.name = name;
    }
}